packages/terra-detail-view/src/DetailView.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import DetailViewSubtitle from './DetailViewSubtitle';
+import '../src/DetailView.scss';
```

Comment:
babel has the flag ` --copy-files` which will copy all of our non-js files over to the lib directory.

https://github.com/cerner/terra-ui/pull/122/files#diff-65401bfdac0e3ddb996b953c98ba6cf7R27